src/main/features/response/views/confirmation.njk

```diff
@@ -34,11 +34,12 @@
         <p>{{ t('You can avoid getting a County Court Judgment if the claimant accepts your offer.') }}</p>
       {% endif %}
 
-      <h2 class="heading-medium">{{ t('Mediation') }}</h2>
-      <p>{{ t('You have chosen to use the free mediation service to resolve the dispute with {{ claimant }}. We’ll ask them if they want to mediate.', { claimant: claim.claimData.claimant.name  }) }}</p>
-      <p>{{ t('If they agree to mediation we’ll email you to arrange an appointment.') }}</p>
-      <p>{{ t('If you can reach an agreement with {{ claimant }} you won’t need to go to a hearing.', { claimant: claim.claimData.claimant.name  }) }}</p>
-
+      {% if claim.response.freeMediation === 'yes' %}
```

Comment:
can you please use the YesNo.Yes option? 
same as here: 
https://github.com/hmcts/cmc-citizen-frontend/blob/master/src/main/features/claim/views/check-and-send.njk#L57